docs/store.js

```diff
@@ -424,7 +413,7 @@ var LocalCalendarStorage = function(params) {
             self.endWork();
         };
         
-        var randomNumber = Math.random() * 4000;
+        var randomNumber = Math.random() * 100;
```

Comment:
As a general rule, hardcoding numbers into the middle of your code makes it difficult to update and understand later. By parameterizing it you can play with the value without rewriting everything all the time.